app/qml/FeatureForm.qml

```diff
@@ -441,16 +446,39 @@ Item {
         }
 
         Label {
-          id: constraintDescriptionLabel
+          id: fieldHelperText
```

Comment:
where is this used?

Reply:
Line 428 https://github.com/lutraconsulting/input/pull/1500/files/dd4b3d3b4c45b52363c6dc486b01d60d4baa08f2#diff-055efeb3c73beb5496dc4f32dd5e5467b2e76e572317083b7d788449598bd5a7R428